Regional chapter policy (#118)
OWASP/www-policy: operational/chapters.md

---

title: Chapters Policy - Draft (WIP)
layout: col-document
document: Rules of Procedure
tags: Rules of Procedure
notice: 2022-10-21

---

{% include draft-notice.html %}

This is a DRAFT Chapter policy incorporating improvements to the existing policy as well as incorporating a standardized regional model created by the Chapter Committee after more than a year of consultations with all OWASP regional chapters. Please provide comments on this policy via the [Community Review Process](https://owasp.org/www-policy/operational/community-review-process).

## Overview

Chapters are central to OWASP's mission of building a community of students, appsec professionals, and developers around the world. This policy defines various chapter types, how to start, run, maintain, and deactivate OWASP chapters.

## Definitions

For the purposes of this policy, the words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.

> "Activity" - local chapter activities, such as chapter meetings, activities such as CTFs or hack-a-thons, social gatherings, etc, that are promoted on the OWASP website, free to attend and participate. Compliant local chapter activities do not require pre-approval under the Events policy, and expenses are governed by the Expenses policy.

Reviewer: ```suggestion
"Activity" - local chapter activities, such as chapter meetings, activities such as CTFs or hack-a-thons, social gatherings, etc, that are promoted on the OWASP website, are free to attend and participate. Compliant local chapter activities do not require pre-approval under the Events policy, and expenses are governed by the Expenses policy.
```

"Event(s)" - pre-approved events as governed by the [Events policy](./events), usually distinguished by paid attendance, with a budget, and usually more than four hours in duration. Events are out of scope for this policy.

"City chapter" - a standard chapter type

"Student chapter" - a chapter led by academic faculty members and students

"Regional chapter" - a chapter that encompasses a region, run by or with associated chapters

> "leader" - volunteers who actively organize chapter activities.

Reviewer: Need to be consistent on these, periods or not?

"Deactivation" - clearing leaders and tagging the chapter as inactive, but not deleting any Chapter content.

"Dissolution", "deletion", and "dissolved" - the chapter leadership SHALL be removed and the chapter SHALL be deleted, usually where a new chapter has failed to start within the allotted time or sanctions forces the OWASP Foundation to close the chapter.

"pre-approved" - means a Chapter support ticket MUST be raised with sufficient time and detail for the request to be approved.

> ## Chapter naming

Reviewer: Title caps

OWASP has various chapter types, and each has a naming scheme to aid discovery and reduce confusion. All chapter names MUST be approved by the OWASP Foundation.

> ### City chapters

Reviewer: Title caos

City chapters SHOULD be named "OWASP «city name»". The city name SHOULD be recognizable to mapping software to allow automated geographic searches or discovery of chapters.

City chapter names MUST NOT be a regional or country name unless the city name is the country name (e.g. OWASP Luxembourg).

> ### Student chapters

Reviewer: Title caps

Student chapters SHOULD be named "OWASP «institution name»" or where the institution has different campus locations, "OWASP «institution name» «campus»"

> ### Regional chapters

Reviewer: Title caps

Regional chapters are to be named "OWASP «region name»" representing a larger geographic area with a cohesive community not limited to a specific city. A region might be a state/district/province, a whole country, or a commonly recognized geographic area.

Residents living in the region MUST recognize regional chapter names as commonly associated with the region. The region MUST include the locations being claimed by the Regional Chapter application. Leaders of this region SHOULD agree that there is a regional community around this name.

> Regional chapter names MUST NOT conflict or likely to create confusion with other OWASP entities, events, committees, chapters, and so on.

Reviewer: ```suggestion
Regional chapter names MUST NOT conflict or be likely to create confusion with other OWASP entities, events, committees, chapters, and so on.
```

Regional chapter names of large encompassing regions, such as "OWASP Europe" or "OWASP LATAM", are unlikely to be approved without sufficient justification and community support. All encompassed city, student, and regional chapters MUST agree to the creation of these large encompassing regions.

> ### Naming precedence

Reviewer: Title caps

Chapter applications SHALL be processed in the order received.

Existing chapters have precedence over newer chapters.

If a new chapter has a similar name to an existing chapter, a more specific name SHALL be used to enable geographic searches and reduce confusion. e.g. OWASP Brooklyn wants to be created, assuming the one hour travel policy is applicable, is different from the existing OWASP NYC.

## Requirements

### Open Access and Discoverability

OWASP's mission requires free and open access to free chapter activities. Chapter leaders MUST NOT restrict access to the general public. Activity locations SHOULD be chosen to encourage participation. Activities MUST be published and discoverable from the OWASP website.

> If an academic institution has membership, campus access, or leadership restrictions or additional requirements that contravenes open access, Student chapters MAY apply in writing to the Chapters Committee for an exemption by explaining the restrictions in full.

Reviewer: ```suggestion
If an academic institution has membership, campus access, leadership restrictions, or additional requirements that contravene open access, Student chapters MAY apply in writing to the Chapters Committee for an exemption by explaining the restrictions in full.
```

### Chapter Leaders

Chapter leadership is on a voluntary basis. Leadership is personal and not associated with any organization, company, or employer.

Each Chapter MUST have a minimum of two (2) and a maximum of five (5) Chapter Leaders. Regional chapters can apply for an exemption to create additional leaders if they have multiple locations.

Chapter leaders MUST be a leader of only one chapter.

> Chapter leaders SHOULD be present in the country of their chapter. Regional chapters leaders MUST reside within the region.

Reviewer: ```suggestion
Chapter leaders SHOULD be present in the country of their chapter. Regional chapter leaders MUST reside within the region.
```

> Chapters with less than two leaders SHALL be deactivated if they do not find sufficient leaders within 90 days.

Reviewer: Do we have any idea how many chapters this might impact upon ratification?

### Student Chapter Leaders

Students and faculty of institutions of higher education MAY create OWASP Student chapters associated with their educational institution, and optionally campus location.

> Student chapters MUST have at least one ongoing faculty member, and at least one student leader. The Faculty member will be representing the institution, and be responsible for ensuring that OWASP policy is followed.

Reviewer: ```suggestion
Student chapters MUST have at least one ongoing faculty member, and at least one student leader. The Faculty member will be representing the institution, and be responsible for ensuring that OWASP policies are followed.
```

> Student Chapters are exempted from meeting minimum leader number requirements between academic years. Student Chapters have 90 days after the start of each new academic year to find at least one student leader.

Reviewer: Is this going to create an oversight nightmare for the foundation?

If there are overlapping regions, the new Regional Chapter SHALL be the smallest of the overlapping regions that incorporate all associated locations.

Regional chapters do not need to have participating city chapters, but MUST meet all activity requirements. Regional chapters can hold activities in multiple locations.

The existence of a regional chapter does not conflict with a City chapter co-located in the same region.

### Chapter Distance

Chapters that are too close together reduce community networking opportunities and increase administrative overheads for chapter leaders, the OWASP Foundation, and the Chapter Committee alike.

The following exceptions are permitted:

New chapters within 80 km or 50 miles of another chapter MAY be formed if more than one hour travel time away from the nearest similar chapter.

Multiple student chapters within the same institution MAY exist in a single city or region but MUST be unique per campus location.

Chapters requiring an exception SHALL be processed on a case-by-case basis by the OWASP Foundation.

### Chapter Communications

> Chapter leadership contact information MUST be accurate and appear on the owasp.org web site. The contact information SHOULD allow participants, Members, the Chapter Committee, or the OWASP Foundation to contact leadership via email. If another contact method is used, it MUST NOT rely upon membership of any non-OWASP platform.

Reviewer: ```suggestion
Chapter leadership contact information MUST be accurate and appear on the owasp.org web site. The contact information SHOULD allow participants, members, the Chapter Committee, or the OWASP Foundation to contact leadership via email. If another contact method is used, it MUST NOT rely upon membership of any non-OWASP platform.
```

Chapter Leaders MUST monitor or forward their owasp.org email address for OWASP or chapter related correspondence from participants, speakers, members, Chapter Committee, the OWASP Foundation and its systems, and the Global Board.

Chapter leadership SHOULD respond in a reasonable time to Member, Chapter Committee, or Foundation communications, and MUST respond within 60 days for the Chapter to be considered active.

Leaders SHOULD reply from their owasp.org email address. Leaders MAY forward their owasp.org email to a personal email address and respond from it to assist with timely communications. Leaders MUST NOT use a corporate email address to avoid any perception or actuality of conflict of interest.

> All supports requests MUST be logged through the OWASP Foundation ticketing system.

Reviewer: ```suggestion
All support requests MUST be logged through the OWASP Foundation ticketing system.
```

### Chapter Branding Requirements

OWASP Chapters MUST identify as an OWASP chapter in their branding, per the latest published [branding guidelines](./branding).

Chapter Leaders SHOULD regularly review the latest templates and use them to create new materials, and provide them to potential speakers as a template for their presentations.  

### Chapter Vendor Neutrality Requirements

Chapters MUST comply with OWASP's core values of transparency and vendor neutrality. They SHOULD avoid the perception or actuality of a conflict of interest with any arrangements for space, speakers, or sponsors. Speakers SHOULD NOT be making a sales pitch or marketing.

OWASP Leaders SHALL advise speakers to use OWASP templates for presentations, but they are not mandatory for local Chapter activity. Recognizing supporters or sponsors SHOULD be only mentioned at the start and / or the end of the activity.

Where a speaker is clearly breaching this section, Chapter Leaders have the right to discontinue the talk and ask the speaker to leave the activity.

> ### Co-Hosting free activities with other organizations

Reviewer: ```suggestion
### Co-Hosting Free Activities
```

If a chapter wishes to regularly co-host activities with another organization, the chapter MUST ensure that the other organization is aware of and agrees to comply with this OWASP Chapter policy and [Code of Conduct](./code-of-conduct). 

Co-hosted activities MUST:

- Comply with this policy, such as being free and open to attend, and comply with vendor neutrality
- Discoverable by new and existing members and participants by publishing the details on the owasp.org website with sufficient notice to allow promotion and attendance

> - OWASP Branding is clearly visible to activity attendees during discovery, registration, and the activity itself.

Reviewer: ```suggestion
- OWASP Branding is clearly visible to activity attendees during discovery, registration, and the activity itself
```

If OWASP branding for the co-hosted activity is visible on the other organization's website or social media, the other organization MAY be promoted on the OWASP website and the chapter's social media accounts.

Compliant co-hosted activities with non-commercial organizations do not need pre-approval, and count towards Chapter activity requirements. Compliant co-hosted free activities with commercial organizations MUST be pre-approved per the next section, and if approved, count towards Chapter activity requirements.

> ### Co-Hosting paid activities or commercial organizations

Reviewer: Title caps

If a Chapter wants to co-organize a paid event or activity with another organization, particularly commercial organizations, this MUST be pre-approved under the relevant [Events policy](./events) or as a co-marketing or a partnership agreement with the OWASP Foundation through [logging a ticket](https://owasporg.atlassian.net/servicedesk/customer/portal/7/create/83?src=691141514).

Once approved, if any OWASP Chapter activity is being held with another organization, OWASP's branding SHOULD be clearly visible to participants.

## Chapter Lifecycle

### Starting or Restarting a Chapter

Prospective Chapter leaders MUST submit a request through [Contact Us](https://contact.owasp.org/).

Once all applicable naming and policy requirements are met, the OWASP Foundation SHALL create a new chapter.

### Update Chapter Information

In the first 30 days, chapter leaders MUST update their chapter page on the owasp.org website with all relevant details, including leader contact details, regular activity location (if known), and any other necessary information about their chapter.

> ### Hold a first activity

Reviewer: Title caps

In the first 30 days, the new chapter MUST schedule and publish their first activity on the OWASP website, and hold it within the first 90 days.

> ### Creating a new Regional Chapter

Reviewer: Title caps

Regional chapters can be created by submitting a Create Regional Chapter support ticket. After validation, the OWASP Foundation will refer the submission to the Chapter Committee for the next available Chapter Committee meeting.

> The Chapters Committee will review the Regional Chapter's proposal and discuss with relevant active community leaders and existing City Chapters leaders in the same region. There MUST be no objection to creating the Regional Chapter from leaders within this region this Regional Chapter. If the responding leaders of these chapters concur with the Regional Chapter's application AND agree that the Regional Chapter can co-exist, the Regional Chapter can be approved.

Reviewer: ```suggestion
The Chapters Committee will review the Regional Chapter's proposal and discuss with relevant active community leaders and existing City Chapters' leaders in the same region. There MUST be no objection to creating the Regional Chapter from leaders within this region. If the responding leaders of these chapters concur with the Regional Chapter's application AND agree that the Regional Chapter can co-exist, the Regional Chapter can be approved.
```

After consultation, the Chapters Committee will vote on the proposed Regional Chapter at the next available Committee meeting.

City Chapters MAY opt to join the Regional Chapter as an associated Chapter or remain an independent overlapping City Chapter (with no connection to the Regional Chapter).

New or existing city chapters cannot be promoted to a regional chapter without prior approval and sufficient justification. This is to deter city chapters going dormant or city leaders trying to claim a region.

> ### Creating a new City Chapter within the region of an overlapping Regional Chapter

Reviewer: Can this be simplified and title cap'ified?

The above will only apply if none of the Regional Chapter Leaders are from the same city as the proposed City Chapter. If leaders of the Regional Chapter reside/work in the same city in which the City Chapter is wanted to be created, and there are multiple activities held in the city, they will be considered the leaders of the City Chapter and will independently reach a decision regarding the creation of this City Chapter.

Note that both the City Chapter and the Regional Chapter MUST continue to meet their individual requirements. In particular, the Regional Chapter MUST still retain consensus from other community members, excluding the city wherein the City chapter exists. The Regional Chapter MUST have consensus from leaders in at least three cities, not including the one where the City Chapter is created.

### Adding, Changing, or Removing Leaders

Existing leaders MUST submit a ticket to the [New Chapter/Leader Request](https://owasporg.atlassian.net/servicedesk/customer/portal/8/group/20/create/105).
If the local chapter leadership is inactive, local chapter members can request new leadership using the same form.

> ### Chapter social media

Reviewer: Title caps
